src/green/UniformDielectric.hpp

```diff
@@ -31,6 +31,7 @@
 
 /*! \file UniformDielectric.hpp */
 
+namespace pcm {
```

Comment:
Thanks:)

Reply:
No problem. I've moved `requirements.txt` to the root because of #104